doc/sphinx/language/core/basic.rst

```diff
@@ -134,8 +134,9 @@ Keywords
   used as identifiers::
 
     _ Axiom CoFixpoint Definition Fixpoint Hypothesis Parameter Prop
-    SProp Set Theorem Type Variable as at cofix discriminated else end
-    fix for forall fun if in let match return then where with
+    SProp Set Theorem Type Variable as at cofix else
+    end fix for forall fun if in let match
+    return then where with
 
   Note that notations and plugins may define additional keywords.
```

Comment:
We should clarify "(including in the prelude)".

Reply:
And define "prelude".